package.json

```diff
@@ -38,12 +38,13 @@
   "author": "netroy <aditya@netroy.in> (http://netroy.in/)",
   "license": "MIT",
   "devDependencies": {
-    "escomplex-js": "^1.2.0",
-    "expect.js": "^0.3.1",
-    "glob": "^6.0.1",
-    "istanbul": "^0.4.0",
-    "jshint": "^2.8.0",
-    "mocha": "^2.3.4",
-    "sinon": "^1.17.2"
+    "escomplex-js": "1.2.0",
```

Comment:
At least, creating a separate pull request for 9dfb252acb4ea0c7200bf9fab756fe334e590cc3 seems much cleaner, because 9dfb252acb4ea0c7200bf9fab756fe334e590cc3 is not directly related to the main purpose of #83.